hugo/content/contact.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Contact
+toc: true
+---
+
+# Contact
+
+You can contact us in several ways:
+
+## Community Channels
+
+* Our [issue tracker on GitHub](https://github.com/mumble-voip/mumble/issues)
+* [Discussions on GitHub](https://github.com/mumble-voip/mumble/discussions)
+* IRC: [#mumble on irc.freenode.net](irc://irc.freenode.org/mumble)
+* Chat: [Mumble room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
+* Our [Forums](https://forums.mumble.info/)
```

Comment:
```suggestion
* [Forums](https://forums.mumble.info/)
```

Reply:
1. Should we use `Forum` instead?
2. Should I add `Official Forum`?

Reply:
All of these are official resources. I think just Forum or Forums is fine.